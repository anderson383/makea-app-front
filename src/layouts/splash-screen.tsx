import { LogoIcon } from "../../assets/icons/logo";
import { THEME } from "../theme";
import { View, Text, StyleSheet } from "react-native"
import { SvgXml } from "react-native-svg"

export const SplashCustomScreen = () => {
  
  return (
    <View style={styles.splash}>
      <SvgXml width={300} xml={LogoIcon} />
    </View>
  )
}

const styles = StyleSheet.create({
  splash: {
    flex: 1,
    backgroundColor: THEME.colors.secondaryColor,
    alignItems: 'center',
    justifyContent: 'center'
  }
});