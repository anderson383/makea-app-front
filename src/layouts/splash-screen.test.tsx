
import {SplashCustomScreen} from './splash-screen'
import {render, fireEvent, waitFor} from '@testing-library/react-native'

describe('Test Splash Screen Component', () => {
  
  const setup = async () => {
    return await waitFor(() => render(
      <SplashCustomScreen />
    )) 
  }


  test(`Should render component`, async () => {
    const {toJSON} = await setup()

    expect(toJSON).toMatchSnapshot()
  })
})