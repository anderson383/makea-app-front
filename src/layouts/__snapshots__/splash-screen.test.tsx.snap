// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Splash Screen Component Should render component 1`] = `[Function]`;
